### ISSUE TITLE
Hard coded references to mysqueezebox.com

### ISSUE THREAD

**Issue by gordonb3** [ralph-irving/squeezeplay#20]
Investigating an LMS issue with corrupted network frames I noticed that my players were continuously trying to target amazonaws servers which appears to be the hosting party for mysqueezebox.com. As far as I can see these references are restricted to the LUA part of squeezeplay OS and I have been able to silence this traffic by commenting out a few lines on my players. I'm not real sure what this will do on players that do not have another server configured though, nor a spare unit for testing. If you like I can submit a pull request for the change I made, but I assume this already has your attention?

**Comment by gordonb3** [comment]
Okay, I'll create a fork and have that ready for you. May want to consider releasing that earlier though, as after closing the firewall on mysqueezebox.com I saw the players continuously re-attempting connect. I have not tried what happens if I set DROP rather than REJECT in the firewall. Possible manifestations could be stream interruptions or the player to become unresponsive to commands.

**Comment by ralph-irving** [comment]
I have uploaded mac and windows test releases with PR#21 applied and they are available at https://sourceforge.net/projects/lmsclients/files/squeezeplay/nomysb/
I changed the base version number to 8.4.1 so the removal of mysb can be identified for applets and programs.
I also replicated the file structure for the imageviewer subfolders to my sdf.org site as it still supports http without redirecting to https but have not updated the urls in the 8.4.1r1468 build on sourceforge yet as I need to investigate what the usage SLA is for sdf.org first.
I urls in the .lst files have also been changed to point to my site.  Here are the top level urls if you'd like to test it out.

```
http://ralph.irving.sdf.org/static/images/imageviewer/sbtouch.lst
http://ralph.irving.sdf.org/static/images/imageviewer/sbradio.lst
http://ralph.irving.sdf.org/static/images/imageviewer/sbcontroller.lst
```
